opencda/scenario_testing/config_yaml/v2xp_datadump_town06_carla.yaml

```diff
@@ -22,6 +22,16 @@ world:
     fog_falloff: 0 # Density of the fog (as in specific mass) from 0 to infinity. The bigger the value, the more dense and heavy it will be, and the fog will reach smaller heights
     wetness: 0
 
+bp_meta_path: "/home/hao/code/python/bbx_stats.json"
+use_multi_class_bp: true
```

Comment:
it seems these attributes do not belong to any category (for example, weather belongs to world category), add an overall category to here for better structure

Reply:
structured